modules/@apostrophecms/area/ui/apos/components/AposAreaMenu.vue

```diff
@@ -122,17 +123,25 @@ export default {
         iconOnly: true,
         icon: 'plus-icon',
         type: 'primary',
-        modifiers: [ 'round', 'tiny' ]
+        modifiers: [ 'round', 'tiny' ],
+        iconSize: this.empty ? 20 : 11
       }
     };
   },
   computed: {
     moduleOptions() {
       return window.apos.area;
     },
+    contextOpts() {
```

Comment:
The naming could get confusing. `extendedContextOptions`?

Reply:
changed to the even more verbose `extendedContextMenuOptions`